client/blocks/upsell-nudge/style.scss

```diff
@@ -48,7 +48,7 @@
 		color: var( --color-text-inverted );
 	}
 
-	.banner__title {
+	.banner__info .banner__title {
```

Comment:
As a minor note I'd recommend writing this out in longform when the file gets too long. This is more of a stylistic preference, so use your best judgement on when to use it.

For example in this diff one might assume that the selector here is `.banner__info .banner__title` but it's actually `.upsell-nudge .banner__info .banner__title` with a specificity of (0, 3, 0 ) instead of (0, 2, 0)